docs/documentation/connect.rst

```diff
@@ -0,0 +1,988 @@
+Kafka Connect
+=============
+
+.. contents::
+    :local:
```

Comment:
Do we want to limit these local TOCs? At current depth this sometimes pushes the first content below the fold for many laptop users and I'm not sure how much value we get past the first level now that these are broken up topically.

Reply:
@ewencp i don't think we should, unless the AK site will support left-hand TOC expansion (like CP docs). is that going to happen?

Reply:
I was avoiding doing the left-hand TOC expansion for now since it requires more styling. It's absolutely possible to do though. In this case though, the limits I meant were just to do, e.g., 2 levels. The current version looks like this:

![screen shot 2018-02-28 at 10 16 19 am](https://user-images.githubusercontent.com/47366/36804816-768eba4c-1c70-11e8-9213-c4a160ad8fc7.png)

I was just questioning whether, e.g., the individual sections 3 levels deep under, e.g., "Developing a Simple Connector" are that valuable. Another way of putting it is that a section that deep in the page is individually useful, maybe we want to split this up further (which is perhaps related to my other comment about preemptively providing a `connect/` directory even if we start out only with `index.html`).